### ISSUE TITLE
Increased Triceps Skinfold thickness

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5895]
**Preferred term label:** Increased Triceps Skinfold thickness

**Synonyms** Increased TSF

**Definition (free text, please give PubMed ID)** Skinfold measured halfway down the back of the upper arm is increased. This is a way to estimate total body fat percentage., PMID: 27669294

**Parent term (use hpo.jax.org/app)** Increased adipose tissue HP:0009126

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding term
